src/web_sys.rs

```diff
@@ -387,6 +387,32 @@ impl Context {
             }
         }
     }
+
+    pub unsafe fn tex_sub_image_2d_with_html_image(
+        &self,
+        target: u32,
+        level: i32,
+        x_offset: i32,
+        y_offset: i32,
+        format: u32,
+        ty: u32,
+        image: &HtmlImageElement,
+    ) {
+        match self.raw {
+            RawRenderingContext::WebGl1(ref gl) => {
+                gl.tex_sub_image_2d_with_u32_and_u32_and_image(
+                    target, level, x_offset, y_offset, format, ty, image,
+                )
+                .unwrap(); // TODO: Handle return value?
+            }
+            RawRenderingContext::WebGl2(ref gl) => {
+                gl.tex_sub_image_2d_with_i32_and_i32_and_u32_and_type_and_html_image_element(
```

Comment:
Should we use [`tex_sub_image_2d_with_u32_and_u32_and_html_image_element`](https://docs.rs/web-sys/0.3.45/web_sys/struct.WebGl2RenderingContext.html#method.tex_sub_image_2d_with_u32_and_u32_and_html_image_element) to match WebGL1 here, instead of determining `width()` and `height()` from `image`?

Reply:
Ah yes, absolutely. I missed that in the forest of overloads :)